packages/daemon/src/__tests__/README.md

```diff
@@ -0,0 +1,3 @@
+# Tests
+
+Please place your tests in this folder.
```

Comment:
It's good to describe the test directory here, just like this page: https://github.com/nodejs/node/tree/master/test#nodejs-core-tests

Reply:
It's helpful for the contributors, I will supplement this document when refactoring the unit tests.